docs/guides/Package-authoring-best-practices.md

```diff
@@ -0,0 +1,179 @@
+>A copyright notice essentially indicates that your work cannot be copied without your permission. Including a copyright notice in your package is easy and won't do any harm!
+
+Example: Copyright (c) Contoso 2020
+
+### Licensing
+
+✔️ DO [include a license expression or license file in your package](https://docs.microsoft.com/nuget/reference/msbuild-targets#packing-a-license-expression-or-a-license-file).
+> [!IMPORTANT]
+> A project without a license defaults to [exclusive copyright](https://choosealicense.com/no-permission/), meaning that you have not granted anyone permission to use your project.
+
+❌ DO NOT use the deprecated `LicenseUrl` metadata property.
+> This presents legal ambiguity as license changes at the URL will retroactively change the displayed license for previous package versions.
+
+#### If your package is [open source](https://opensource.org/osd)
+
+✔️ DO [choose an open source license](https://choosealicense.com/) to make your package open source.
+> *"Open source licenses are licenses that comply with the Open Source Definition — in brief, they allow software to be freely used, modified, and shared."* - Open Source Initiative. To learn more about open source software and the Open Source Initiative, check out https://opensource.org/.
+
+✔️ CONSIDER [including a license expression in your package](https://docs.microsoft.com/nuget/reference/msbuild-targets#packing-a-license-expression-or-a-license-file).
+> License expressions are surfaced the most clearly and make it more obvious to consumers if they can use your package or if the license has changed. 
+> [!Note]
+> NuGet.org only accepts license expressions for licenses that are approved by the Open Source Initiative or the Free Software Foundation.
+
+#### If your package is not open source
+
+✔️ DO [include a license file in your package](https://docs.microsoft.com/nuget/reference/msbuild-targets#packing-a-license-expression-or-a-license-file).
+> Any license file (.txt or .md) can be added to your package, including non-standard licenses. 
+
+### Project URL
+
+✔️ CONSIDER including a link to an associated project, repository, or company website.
+> Your project site should have everything users need to know about your package and will likely be where users look for documentation.
+
+### Icon
+
+✔️ CONSIDER including an icon with your package to help visually differentiate it. It's a relatively small addition that can improve perception of package quality.
+> Icons can be specific to individual packages or be a brand logo.
+
+✔️ DO use an image that is 128x128 and has a transparent background (PNG) for best viewing results.
+> NuGet will automatically scale your image to the client it is being displayed on.
+
+❌ DO NOT use the deprecated `IconUrl` metadata property.
+
+### Repository Type and URL
+
+✔️ CONSIDER including a repository type and URL to make it easy for package consumers to view source code, report issues, and view repository documentation.
```

Comment:
Should we put the Source Link paragraph first as it is the best way to fill out this info? I don't feel strongly about this, feel free to keep it as is if you prefer it :)